model/host/host.go

```diff
@@ -1778,6 +1788,57 @@ func (h *Host) SetInstanceType(instanceType string) error {
 	return nil
 }
 
+func AggregateSpawnhostData() (*SpawnHostUsage, error) {
+	res := []SpawnHostUsage{}
+	pipeline := []bson.M{
+		{"$match": bson.M{
+			StartedByKey: bson.M{"$exists": true, "$ne": ""},
+			StatusKey:    bson.M{"$in": evergreen.UpHostStatus},
+		}},
+		{"$group": bson.M{
+			"_id":         nil,
+			"hosts":       bson.M{"$sum": 1},
+			"unexpirable": bson.M{"$sum": bson.M{"$cond": []interface{}{"$" + NoExpirationKey, 1, 0}}},
```

Comment:
Do you mean instead of the []interface{}? []string doesn't work, since the NoExpiration field is a boolean. I'm switching it to True/False for readability.